lib/builder/webpack/base.js

```diff
@@ -91,7 +91,12 @@ export default class WebpackBaseConfig {
       chunkFilename: this.getFileName('chunk'),
       publicPath: isUrl(this.options.build.publicPath)
         ? this.options.build.publicPath
-        : urlJoin(this.options.router.base, this.options.build.publicPath)
+        : urlJoin(
+          this.options.router.base,
+          this.options.router.base !== '/'
```

Comment:
Could you explain more on this?
If base is /app and public path is /nuxt, then result is/app/.nuxt ?